src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +17,6 @@
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
It's good practice to handle exceptions that may be thrown when waiting for all futures to complete, as well as during the shutdown of the executor. Consider wrapping `CompletableFuture.allOf(futures).join()` and `executor.shutdown()` in a try-catch block to handle possible interruptions and ensure proper shutdown of the executor.

Reply:
not a terrible idea, but up to you